guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -91,7 +91,13 @@ Ensure {SmartProxy} has access to `{RepoRHEL8ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteMaintenanceProductVersion}
```

Comment:
If we do `self-upgrade` (after enabling RepoRHEL8ServerSatelliteCapsuleProductVersion below!) this is not required, as `self-upgrade` will do it itself.

Reply:
Okay. Removed above command :+1: